zaza/openstack/charm_tests/kerberos/setup.py

```diff
@@ -0,0 +1,239 @@
+# Copyright 2020 Canonical Ltd.
```

Comment:
As discussed 1:1, several methods in zaza/openstack/charm_tests/kerberos/setup.py feels like code that is on the wrong side of the charm boundary, and should not typically be inside the test code.  My view is that the test fixture charm should typically be handling a lot of that functionality.

In this case, as a tactical measure for this tricky test fixture, we've agreed to go forward as-is. Otherwise, a fair amount of additional test fixture development time, including layers and interfaces, would be necessary to avoid the boundary concerns.